### PR TITLE
example for electron react app with portals

### PR DESCRIPTION
**Pull request by kunalsinghal**

### CODE REVIEW

File: examples/electron/index.html [hunk 1]
```diff
@@ -73,13 +73,11 @@ <h3>The basics</h3>
       </p>
     </div>
 
-    <script src="https://unpkg.com/@grammarly/editor-sdk?clientId=YOUR_CLIENT_ID"></script>
+    <script src="https://unpkg.com/@grammarly/editor-sdk?clientId=client_JHPHiNzDfEDjzBTLs6hMzJ"></script>
```
Comment (ljhaywar) [inline examples/electron/index.html:76]: Client id's slipped into this file

File: examples/electron-react-portals/main.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    },
+  });
+
+  win.loadURL("file://" + path.join(__dirname, "build/index.html"));
+
+  return win;
+};
+
+app.setAsDefaultProtocolClient("example");
+function enableTemporaryGrammarlyOriginWorkaround() {
```
Comment (ljhaywar) [inline examples/electron-react-portals/main.js:20]: I don't think we need this anymore

File: examples/electron/index.html (hunk 1 above)
Comment (AndrewDiMola) [reply examples/electron/index.html:76]: no longer an issue.

**Comment by AndrewDiMola** [comment]
Closing for now; may reopen to add a similar example in the future.

**Pull request closed without merging**